Author: Hiltjo Posthuma <hiltjo@codemadness.org>
Date:   Fri, 19 Mar 2021 11:25:29 +0100

update static bearer token

This uses the same bearer as youtube-dl.

Thanks frobnitz!

1 file changed, 1 insertion(+), 1 deletion(-)
diff --git tscrape_update.orig tscrape_update
@@ -10,7 +10,7 @@ tscrapepath="$HOME/.tscrape/feeds"
 maxjobs=8

 # Twitter authentication bearer (seems to be static).
-bearer="AAAAAAAAAAAAAAAAAAAAANRILgAAAAAAnNwIzUejRCOuH5E6I8xnZz4puTs%3D1Zv7ttfk8LF81IUq16cHjhLTvJu4FA33AGWWjCpTnA"
+bearer="AAAAAAAAAAAAAAAAAAAAAPYXBAAAAAAACLXUNDekMxqa8h%2F40K4moUkGsoc%3DTYfbDKbT3jJPCEVnMYqilB28NHfOPqkca3qaAxGfsyKCs0wRbw"

 # guest token.
 token=""
